go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/prometheus/client_golang v1.6.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/spf13/pflag v1.0.5
-	github.com/tektoncd/pipeline v0.17.1
+	github.com/tektoncd/pipeline v0.17.1-0.20201006183654-d5df1c164a48
```

Comment:
This took me a little time to understand, is so weird what Tekton did, the `d5df1c16` commit was merged since v0.17.1 tag but only available starting at v0.18.0 tag. I think by doing this fixed commit we only lose like 3 commits from the original v0.17.1 tag which are just around docs. I think once this PR is merged, I will bump tekton to v0.18.0 asap